clickhouse-native-jdbc/src/test/java/com/github/housepower/jdbc/AbstractITest.java

```diff
@@ -30,6 +30,8 @@
     protected static final ZoneId SERVER_TZ = ZoneId.of("UTC");
     protected static final String DRIVER_CLASS_NAME = "com.github.housepower.jdbc.ClickHouseDriver";
     protected static final int SERVER_PORT = Integer.parseInt(System.getProperty("CLICK_HOUSE_SERVER_PORT", "9000"));
+    protected static final String SERVER_USER = System.getProperty("CLICK_HOUSE_SERVER_USER", null);
```

Comment:
default user should be 'default' instead of null.

Reply:
default password can be empty string.

Reply:
I'll change that

Reply:
the default password can be empty string too,   ""